index.js

```diff
@@ -65,6 +66,19 @@ function fetch (iri, options) {
         }))
       })
     })
+  } else if (options.method === 'HEAD') {
+    return new Promise((resolve) => {
+      const exists = fs.existsSync(pathname)
```

Comment:
Maybe use `promisify(fs.access)(file, fs.constants.R_OK)`? Would match better with the semantics of `GET`, where `404` is used for any error from the stream. Also we would stay async on IO operations.

Reply:
Since I was making the change, and since async/await is supported in the LTS versions, I went ahead and switched to that syntax if you don't mind. In doing so for the test files, I noticed that there were two tests that weren't set up correctly as they would always pass despite not passing `utf8` to `readFileSync`.

I've also fixed the "standard" errors.